public_html/wp-content/themes/pattern-directory/src/store/utils.js

```diff
@@ -4,6 +4,9 @@
 import { addQueryArgs } from '@wordpress/url';
 import { __ } from '@wordpress/i18n';
 
+// Number of patterns per page, should be multiple of 2 and 3 (for 2- and 3-column layouts).
+export const PER_PAGE = 18;
```

Comment:
Is this a default that can be added to the custom post type instead of here?

Reply:
Hm, I think there was a reason I wanted to define it here, but I'm not sure what that was 🙃 

Changing the default in for the CPT is a better idea

Reply:
Done in https://github.com/WordPress/pattern-directory/pull/110/commits/fe84f2445ca34ce6a5043996c48fdbcc08cc344d - any request that doesn't specify a per_page will use the default of 18.